### ISSUE TITLE
External Monitor is not recognized; used a VGA to HDMI Adapter 

### ISSUE THREAD

**Issue by nonspsb** [emoacht/Monitorian#203]
I recently got a new laptop. However, it has no VGA port. My external monitor is kind of old (HP LE1901wm) and it can only be connected through VGA. I used a VGA to HDMI adapter and probe.log seems to have recognized my external monitor but the application itself won't show the brightness control for my external monitor.

It worked on my old laptop which had a VGA port.

![image](https://user-images.githubusercontent.com/87975500/126972229-d43c3d12-feb2-49d1-b840-a47b8242ba9d.png)
![image](https://user-images.githubusercontent.com/87975500/126972734-ea83c608-9c9b-4ce1-8e97-62f3d0a25b47.png)

**Comment by nonspsb** [comment]
It seems like the VGA to HDMI Adapter was not compatible but I'm not really sure. I bought a DVI to HDMI cable and it worked! 

![image](https://user-images.githubusercontent.com/87975500/127270325-bdda9004-d32a-4283-9072-e75d19b70201.png)

**Comment by emoacht** [comment]
FYI, the controllability of external monitors mainly depend on the results of "Monitor Configuration" functions. Clipping probe.log will make it less useful.

**Comment by lfrohling** [comment]
I also have this problem - I have a USB to VGA/HDMI converter (I need my USB C to connect my Quest 2 and my HDMI connection is already being used for another external monitor). I noticed that when I connect from USB 3 (not type C, the standard USB type just 3.0) I can't control the monitor if I use the VGA output of the converter. But if I use the HDMI output (I believe) I can still change the brightness. Yes I could change VGA to HDMI but that would mean getting behind the computer and into a rats nest of wires each time I want to plug in the Quest 2.

Further I noticed my screen going "black" every two minutes or so for a second or two when it was connected via HDMI (via converter) and so I'd prefer to stick with the stabler VGA (same resolution /output anyway).

Any chance of updating the code so it can be "read"/passed/coded/sent through a VGA cable as well? I have no doubt the big brains you all have for allowing such manipulation through HDMI (god bless you all for this program! It's been sooooo helpful through the years!!) in the first place...?

Does flattery still get people everywhere...?
